doc/extending/extending_theano.txt

```diff
@@ -880,7 +880,14 @@ Documentation
 -------------
 
 See :ref:`metadocumentation`, for some information on how to generate
-the documentation.
+the documentation. The docstrings of all the classes and functions respect the
```

Comment:
I forgot about PEP8 and there are a few changes I'd like to add. 
Please correct this section as follows:

1. Put every point in a bullet list (Note that I am adding 2 new points and moving the previous ones around):
    - "The code should be compatible with Python 2.6 and above, as well as Python 3.3 and above (using 2to3 for conversion to Python 3.x)."
    - "All the code should respect the [PEP8 Code Style Guide](http://www.python.org/dev/peps/pep-0008)"
    - "The docstrings of all the classes .."
    - "To cross-reference other ..."
    - "See :ref:`metadocumentation` ..."
2. Fix this sentence "The docstrings of all the classes and functions **should** respect the .."
3. Fix this "You can use the elemwise op" --> ```You can use the :class:`~theano.tensor.elemwise.Elemwise` Op```
4. Modify the docstring of theano.misc.doubleop.DoubleOp as in line 3
5. Add `doc/library/tensor/elemwise.txt`, with this content
```
===================================================================
:mod:`tensor.elemwise` --  Tensor Elemwise
===================================================================

.. testsetup::

   from theano.tensor.elemwise import *

.. module:: tensor.elemwise
   :platform: Unix, Windows
   :synopsis: Tensor Elemwise
.. moduleauthor:: LISA

.. automodule:: theano.tensor.elemwise
    :members:
```
4. "...use this syntax" --> "...use the [cross-referencing objects](http://www.sphinx-doc.org/en/stable/markup/inline.html#xref-syntax) syntax. \`\`:py\`\` can be omitted, see e.g. this [stackoverflow answer](http://stackoverflow.com/a/7754189).